README.md

```diff
@@ -217,21 +217,107 @@ Keen.ready(function(){
 
 
-## Query Caching
+## Saved Queries
 
-Data sent to Keen is available for querying almost immediately. For use cases that don’t require up-to-the-second answers but require fast performance, query caching can be used to speed up a query. To include query caching as a feature, just add the `maxAge` query parameter to any other query parameters you’ve already specified. The first time your application makes a query specifying the max_age the answer will be calculated normally before it can be cached for future uses.
+There are multiple ways to interact with the saved queries API.
+
+### Example Usage
 
 ```javascript
-var count = new Keen.Query("count", {
-    eventCollection: "pageviews",
-    groupBy: "property",
-    timeframe: "this_7_days",
-    maxAge: 300 // include maxAge as a query parameter to activate Query Caching
+// Create a client instance
+var client = new Keen({
+  projectId: "YOUR_PROJECT_ID",
+  readKey: "YOUR_READ_KEY"
+});
+
+Keen.ready(function() {
+  var savedQuery = this.client.savedQuery();
+
+  // Get all saved queries in a project
+  savedQuery.all(function(err, res) {
+    if (err) {
+      // there was an error
+    }
+    else {
+      // do something with array of saved queries
+    }
+  });
+
+  // Get individual saved query by name
+  savedQuery.get("saved-query-name", function(err, res) {
+    if (err) {
+      // there was an error
+    }
+    else {
+      // do something with single saved query object
+    }
+  });
+
+  // Create a new saved query
+  savedQuery.create("saved-query-name",
+    {
+      refresh_rate: 0,
+      query: { ... },
+      metadata: { 
+        display_name: "saved query name",
+        ...
+      }
+    },
+    function(err, res) {
+      if (err) {
+        // there was an error
+      }
+      else {
+        // your saved query was created successfully!
+      }
+    }
+  );
+
+  // Update an existing saved query
+  savedQuery.update("saved-query-name",
+    { refresh_rate: 86400 },
+    function(err, res) {
+      if (err) {
+        // there was an error
+      }
+      else {
+        // your saved query was updated successfully!
+      }
+    }
+  );
+
+  // Delete an existing saved query
+  savedQuery.destroy("saved-query-name", function(err, res) {
+    if (err) {
+      // there was an error
+    }
+    else {
+      // your saved query was deleted successfully!
+    }
+  });
 });
 ```
-`maxAge` is an integer which represents seconds. The maximum value for `maxAge` is 129600 seconds or 36 hours. Read more about Query Caching in the Keen IO [Data Analysis Docs](https://keen.io/docs/data-analysis/caching/).
 
-**Tip:** If you want to speed up your queries but maintain freshness, you can cache a year-long query and combine the result with a normal query that calculates the most current day’s answer.
+If you only want to see the query results of a saved query, you can use `client.run` by passing in the query name.
```

Comment:
Just wanted to say this is such a great design.. CRUD interface for managing these things, and extending `.run()` for grabbing results. ^5s